.github/workflows/nightly-test.yml

```diff
@@ -0,0 +1,47 @@
+name: TileDB Nightly Test Build
+
+on:
+  schedule:
+    # runs every day at 3:00 UTC
+    - cron: "0 3 * * *"
```

Comment:
There is a hint in their docs that load peaks at 'round' times.  So I tend to take a minute value away from 0.